src/rules/no-role-presentation-on-focusable.ts

```diff
@@ -0,0 +1,39 @@
+import type { Rule } from "eslint";
+// import type { AST } from "vue-eslint-parser";
```

Comment:
it would be better if we removed the comments, no?

Reply:
done